Documentation/special-services/pod-mutator.md

```diff
@@ -0,0 +1,76 @@
+# PodMutator Service
+To provide pod mutation capabilities for finer-grained control over rkt containerization concepts and debugging needs, 
+this proposal introduces a new optional stage1 app/service with special semantics that is lightweight.
```

Comment:
I agree with @yifan-gu's comments and also we can remove this with ease in the future, as opposed to deprecating API commands which needs to supported for a long while.